### ISSUE TITLE
'esbuild' is not recognized as an internal or external command

### ISSUE THREAD

**Issue by jongcodes** [firebase/firebase-tools#7549]
### [REQUIRED] Environment info

**firebase-tools: 13.15.1**

**Platform: Windows**

### [REQUIRED] Test case

I am using the Web Frameworks Integrate for working with Next.js.

### [REQUIRED] Steps to reproduce

I executed the ```firebase deploy``` command.

### [REQUIRED] Expected behavior

The deployment process should not have any errors, but I found an error marked in red.

### [REQUIRED] Actual behavior

![Screenshot 2024-08-11 193300](https://github.com/user-attachments/assets/f1092849-15ff-41d6-8ad4-c54db55fe5eb)

```
'esbuild' is not recognized as an internal or external command,
operable program or batch file.
Failed to get global esbuild version: Error: Command failed: esbuild --version
'esbuild' is not recognized as an internal or external command,
operable program or batch file.
```

To resolve the above error, I ran the command ```npm install -g esbuild```, which successfully fixed the issue.

Since the Firebase documentation does not mention installing esbuild, I believe esbuild should be bundled with the firebase-tools installation.

In my opinion, this error is likely linked to the second point outlined in the release notes below.

https://github.com/firebase/firebase-tools/releases/tag/v13.15.1

**Comment by aalej** [comment]
Hey @jongcodes, thanks for reporting this and for sharing your observations. I was able to reproduce the behavior you mentioned. Sharing the [mcve](https://github.com/aalej/issues-7549) here. I’ll raise this to our team, and mark this issue as reproducible.

Also, from what I can tell, deployments are still successful even if this error message is shown. Just to verify, is this also similar to what you observed, or does the error block deployments?

**Comment by leoortizz** [comment]
Hey @jongcodes, a fix was implemented in #7555 and will be available in the next CLI release
